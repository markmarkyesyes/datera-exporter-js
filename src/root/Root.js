import React, { Component } from "react";
import { hot } from "react-hot-loader";
import "./Root.css";

class Root extends Component {
    render() {
        return (
            <div className="Root">
                <h1> Hello, World! </h1>
            </div>
        );
    }
}

export default hot(module)(Root);