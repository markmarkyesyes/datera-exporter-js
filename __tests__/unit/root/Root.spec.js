import React from 'react';
import Root from '../../../src/root/Root';
import renderer from 'react-test-renderer';

test('tests implementation', () => {
    const tree = renderer
        .create(<Root />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})