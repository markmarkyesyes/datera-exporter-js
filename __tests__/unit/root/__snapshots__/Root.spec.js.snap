// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests implementation 1`] = `
<div
  className="Root"
>
  <h1>
     Hello, World! 
  </h1>
</div>
`;
